docs/local-setup.md

```diff
@@ -0,0 +1,93 @@
+# Setting up a local development environment
+
+This guide assumes that you are comfortable working with on the
+command line with tools like Git.
+
+There are instructions for building [with Docker](#with-docker) or
+[without Docker](#without-docker). Building with Docker means that
+not all of the related repositories need local clones, and that you
+will be building with a version of PHP that is known to work.
+
+When working with multiple translations, or working on changes that may
+also require changing files in the `doc-base` or `phd` repositories, the
+suggested way to organize the local enviroment is to clone repositories
+into a single `phpdoc` directory, and for individual languages to be cloned
+into directories named `{LANG}` instead of `doc-{LANG}` as they are named
+on GitHub.
+
+<a name="with-docker"></a>
+## Building with make and Docker
+
+- Install Docker (https://docs.docker.com/get-docker/)
+- Clone the `doc-en` Git repository
+- Rebuild the documentation using `make`
+- Open output/php-chunked-xhtml/ in your browser.
+
+```sh
+$ mkdir phpdoc
+$ cd phpdoc
+$ git clone https://github.com/php/doc-en.git en
+$ cd en
+$ make
+$ open output/php-chunked-xhtml/index.html
+```
+
+If the `doc-base` or `phd` repositories are available in directories
+adjacent the clone of the `doc-en` repository, those will be used for
+building, otherwise the latest revision of those repositories from GitHub
+will be built into the Docker image used.
+
+To force the Docker image used for building to itself be rebuilt, run
+`make -B build`, otherwise the `Makefile` will only build it if does not
+already exist.
+
+The web version of the documentation with `make php` and the output will
+be placed in `output/php-web`. (See the [additional local web setup
+instructions](local-web-setup) for details on how to view those.)
+
+<a name="without-docker"></a>
+## Building without make or Docker
+
+## Check out the PHP documentation using Git
+
+Note that `doc-en` is cloned into the `en` directory below.
+
+```
+$ mkdir phpdoc
+$ cd phpdoc
+$ git clone https://github.com/php/phd.git
+$ git clone https://github.com/php/doc-base.git
+$ git clone https://github.com/php/doc-en.git en
+```
+
+## Validate and build `.manual.xml`
+
+```
+$ php doc-base/configure.php
```

Comment:
Same comment as above on this being `doc-en` oriented. An additional sentence and example on how to do the same for translation would probably be helpful.